docs/manual-CN/2.query-language/0.README.md

```diff
@@ -1,3 +1,75 @@
 # 面向的读者
 
-本章介绍 **Nebula Graph** 的查询语言，适合所有使用 **Nebula Graph** 的用户
+本章介绍 **Nebula Graph** 的查询语言，适合所有使用 **Nebula Graph** 的用户。
+
+## 占位标识符和占位符值
+
+**Nebula Graph** 查询语言 nGQL 参照以下标准设计：
+
+- NotationtakenfromISO/IEC10646
+- NotationprovidedinISO/IEC39075
+- ISO/IEC JTC 1 N 14279
+
+在模板代码中，任何非关键词，文字或标点符号的标记均为占位标识符或占位符值。例：
+
+```ngql
```

Comment:
下面的部分都不要了

Reply:
I'll keep some of the examples to make the doc more informative.

Reply:
> I'll keep some of the examples to make the doc more informative.

I DO NOT agree since the examples introduce too much context about nGQL. And will be very hard to understand at this moment.

Reply:
fixed